content/es/become-a-sponsor.md

```diff
@@ -17,12 +17,13 @@ Esto es grande, y nosotros queremos que tú seas parte de esto. Ayudanos abrir e
 
 ## Patrocinio Corporativo
 
-{{< become-a-sponsor
-  platinum="Platino"
-  gold="Oro"
-  gold_basic="Gold Basic"
+<!-- Note for translators: words in quotes need to be translated -->
+{{< become-a-sponsor 
+  platinum="Platino" 
+  gold="Oro" 
+  gold_basic="Gold Basic" 
   silver="Plata"
->}}
+> }}
```

Comment:
```suggestion
>}}
```

The logs of the fail build https://github.com/letsencrypt/website/pull/1292/checks?check_run_id=3404553164 indicate:

> Error: Error building site: "/home/runner/work/website/website/content/es/become-a-sponsor.md:26:1": unrecognized character in shortcode action: U+003E '>'. Note: Parameters with non-alphanumeric args must be quoted

So this should fix it